similar/main/credits.cpp

```diff
@@ -217,6 +217,8 @@ static window_event_result credits_handler(window *, const d_event &event, credi
 			songs_set_volume(GameCfg.MusicVolume);
 			songs_play_song( SONG_TITLE, 1 );
 			std::default_delete<credits>()(cr);
+			if (wind)
```

Comment:
OK, yeah I got a bit confused about some conflicting views in this article [http://stackoverflow.com/questions/6731331/is-it-still-safe-to-delete-nullptr-in-c0x](url) but the overall consensus does seem to be that it's safe. Happy to remove these checks (there were a few).